daml-script/test/src/com/digitalasset/daml/lf/engine/script/test/SingleParticipant.scala

```diff
@@ -329,7 +329,8 @@ case class ScriptExample(dar: Dar[(PackageId, Package)], runner: TestRunner) {
 
 case class TraceOrder(dar: Dar[(PackageId, Package)], runner: TestRunner) {
   val scriptId = Identifier(dar.main._1, QualifiedName.assertFromString("ScriptTest:traceOrder"))
-  def traceMsg(msg: String) = s"""[DA.Internal.Prelude:540]: "$msg""""
+  def traceMsg1(msg: String) = s"""[DA.Internal.Prelude:540]: "$msg""""
+  def traceMsg2(msg: String) = s"""[ScriptTest:540]: "$msg""""
```

Comment:
Not quite sure I understand why the location here has changed. I guess we lifted some part but shouldn’t we still end up somewhere in `DA.Internal.Prelude` in the end?

Reply:
It's a consequence of inlining -- the call has moved into `ScriptTest`. OTOH, the location does not look correct.

Reply:
The locations are a bit bogus anyway so I wouldn’t worry about it for now.